README.md

```diff
@@ -58,9 +59,33 @@ Window {
 
 Requires Qt >= 5.8
 
+## Cargo features
+
+Cargo provides a way to enable (or disable default) optional [features](https://doc.rust-lang.org/cargo/reference/features.html).
+
+### `log`
+
+By default, Qt's logging system is not initialized, and messages from e.g. QML's `console.log` don't go anywhere.
```

Comment:
Don't they go in the stderr by default?

I mean, they follow the default Qt handler which is also quite configurable with env variable: https://doc.qt.io/qt-5/debug.html#warning-and-debugging-messages  /  https://doc.qt.io/qt-5/qloggingcategory.html#configuring-categories

Reply:
Idk, by default they just not showing up at all on my system.